Projekty/Projekt1/VinichenkoYevheniiWolnyKrzysztof/feature_engineering.ipynb

```diff
@@ -0,0 +1,1463 @@
+{
```

Comment:
<p>Zawsze piszcie czy to acc czy coś innego</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>